src/class-fields/zero-zero-initializer-var.case

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: zero initialized zero var
```

Comment:
This should be a single line, human readable sentence that provides a basic explanation of the test itself. (for all case files)